src/Score.php

```diff
@@ -100,6 +100,7 @@ public function validate($aValue) {
                 sprintf("Value must be between %s and %s", $this->min, $this->max)
             );
         }
+        return true;
```

Comment:
I'm okay with this change in the major version, but ultimately it is a public interface change, the expected existing interface was any non-exception is validation. I didn't check for other validation functions, do they match this? Do we have a unit test to confirm the return value here?